packages/react-reconciler/src/ReactFiber.js

```diff
@@ -298,9 +298,12 @@ function createFiberImplObject(
   if (__DEV__) {
     fiber._debugInfo = null;
     fiber._debugOwner = null;
+    if (enableOwnerStacks) {
+      this._debugStack = null;
+      this._debugTask = null;
```

Comment:
Shouldn't this be
```suggestion
      fiber._debugStack = null;
      fiber._debugTask = null;
```
?

Reply:
Oops, my bad.